CameraOverlay/overlay_all_stats.py

```diff
@@ -129,8 +132,19 @@ def on_message(self, client, userdata, msg):
 				self.data["reed_velocity"] = 0
 				self.data["reed_distance"] = 0
 				self.data["count"] = 0
+		
+		elif topic == str(topics.DAShboard.receive_message):
+			message = msg.payload.decode("utf-8")
+
+			# reset counter once message is recieved 
+			self.frame_counter = 0  
 
+			# Display Message
+			self.message_canvas.draw_text(message, (190, self.text_height * 5), size= 1, colour=Colour.red)
```

Comment:
What happens when someone sends a second message before the first one gets cleared?

Reply:
I added code so that the priority is given to the first message received. The second wont be displayed.

Reply:
Maybe for a future PR, but I don't think this is the best solution. Either show both, or queue up the messages so they all get shown. Discarding potentially important messages could be problematic.

Reply:
Maybe for now just show the most recent message so at least we aren't throwing any away